### DISCUSSION TITLE
Nginx and Yggdrasil IPv6 range

### DISCUSSION THREAD

**Discussion by d47081** [yggdrasil-network/yggdrasil-go#1055] [Q&A] [answered]
Hello, 

I've configured web server to accept yggdrasil connections only, according to the range 0200::/7 as provided in [documentation](https://yggdrasil-network.github.io/faq.html#will-yggdrasil-conflict-with-my-network-routing)

But I doubt about remote addresses started from 300, will allow following nginx rule to connect them also?

```
server {

  listen [::]:80 ipv6only=on default_server;

  allow 0200::/7;
  deny  all;

}
```

**Comment by zhoreeq** [comment, marked as answer]
What makes you doubt it? 300::/8 is a part of 200::/7

**Reply by d47081** [reply]
I'm not experienced in IPv6 protocol, so asking to make sure users of this range able to connect

Thank you

**Reply by deavmi** [reply]
Yo @zhoreeq howsit
